packages/f/font-inter-ttf/pspec_x86_64.xml

```diff
@@ -0,0 +1,37 @@
+<PISI>
+    <Source>
+        <Name>font-inter-ttf</Name>
+        <Homepage>https://rsms.me/inter/</Homepage>
+        <Packager>
+            <Name>Denis Garaev</Name>
+            <Email>no.email.set.in.config</Email>
+        </Packager>
+        <License>OFL-1.1</License>
+        <PartOf>desktop.font</PartOf>
+        <Summary xml:lang="en">The Inter font family</Summary>
+        <Description xml:lang="en">Inter is a workhorse of a typeface carefully crafted &amp; designed for a wide range of applications, from detailed user interfaces to marketing &amp; signage.
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>font-inter-ttf</Name>
+        <Summary xml:lang="en">The Inter font family</Summary>
+        <Description xml:lang="en">Inter is a workhorse of a typeface carefully crafted &amp; designed for a wide range of applications, from detailed user interfaces to marketing &amp; signage.
+</Description>
+        <PartOf>desktop.font</PartOf>
+        <Files>
+            <Path fileType="data">/usr/share/fonts/truetype/inter/InterVariable-Italic.ttf</Path>
+            <Path fileType="data">/usr/share/fonts/truetype/inter/InterVariable.ttf</Path>
+            <Path fileType="data">/usr/share/metainfo/inter.metainfo.xml</Path>
+        </Files>
+    </Package>
+    <History>
+        <Update release="1">
+            <Date>2024-06-14</Date>
+            <Version>4.0</Version>
+            <Comment>Packaging update</Comment>
+            <Name>Denis Garaev</Name>
+            <Email>no.email.set.in.config</Email>
```

Comment:
It looks like you don't have an email set in the [packager file](https://help.getsol.us/docs/packaging/prepare-for-packaging#setting-up-the-packager-file). Could you set one, please? Thanks!